README.md

```diff
@@ -394,6 +394,16 @@ The no-proxy list will automatically include entries for localhost.
 
 If you're running a Kubernetes variant, the no-proxy list will automatically include the Kubernetes API server endpoint and other commonly used Kubernetes DNS suffixes to facilitate intra-cluster networking.
 
+#### Metrics settings
+
+By default, Bottlerocket sends anonymous metrics when it boots, and once every six hours.
+This can be disabled by setting `send-metrics` to false.
+Here are the metrics settings:
+
+* `metrics.metrics-url`: The endpoint to which metrics will be sent. The default is `https://metrics.bottlerocket.aws/v1/metics`.
+* `metrics.send-metrics`: Whether or not Bottlerocket will send anonymous metrics.
+* `metrics.service-checks`: A list of systemd services that will be checked to determine whether a host is healthy or not.
```

Comment:
```suggestion
* `metrics.service-checks`: A list of systemd services that will be checked to determine whether a host is healthy.
```
nit: "whether" usually implies "or not".  (same in send-metrics, if you don't hate this suggestion.)